.github/workflows/build-test.yml

```diff
@@ -458,8 +458,8 @@ jobs:
         toolchain: [ "stable", "1.81.0" ] # [stable, 1.81.0, beta, nightly]
```

Comment:
Twelve lines further above, there is a commented out `actions/checkout@v4`. Can you please replace it with `actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # ratchet:actions/checkout@v4`

Reply:
I'll fix this in a follow up and merge the PR as is